src/wcl/rope.h

```diff
@@ -126,6 +126,7 @@ class rope_builder {
 
  public:
   void append(const char* str) { r = r.concat(rope::lit(str)); }
+  void append(std::string str) { r = r.concat(rope::lit(str)); }
```

Comment:
you can move that string here, you might as well delete the `const char*` above when you do that since at's what happens here anyway. Later on we made make an `append_static` as an optimization for string_views  that have a static lifetime...again another thing rust does better...sigh.